src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.Figure;
+import core.basesyntax.entity.Circle;
+import core.basesyntax.entity.IsoscelesTrapezoid;
+import core.basesyntax.entity.Rectangle;
+import core.basesyntax.entity.RightTriangle;
+import core.basesyntax.entity.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAX_VALUE = 100;
+    public static final String DEFAULT_COLOR = "white";
+    public static final int DEFAULT_RADIUS = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Figure figure;
+
+        switch (random.nextInt(5)) {
+            case 0:
+                figure = new Circle(
+                        colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE));
+                return figure;
```

Comment:
```suggestion
                return new Circle(
                        colorSupplier.getRandomColor(), random.nextInt(MAX_VALUE));
```
do it everywhere else